src/Joonasw.AspNetCore.SecurityHeaders/XContentTypeOptions/XContentTypeOptionsOptions.cs

```diff
@@ -0,0 +1,25 @@
+﻿namespace Joonasw.AspNetCore.SecurityHeaders.XContentTypeOptions
+{
+    public class XContentTypeOptionsOptions
+    {
+        public XContentTypeOptionsOptions()
+        {
+            
+        }
+
+        /// <summary>
+        /// Defines the X-Content-Type-Options header and whether to allow sniffing
+        /// NoSniff is turned on by default
```

Comment:
The comment says it is enabled by default, but this is the constructor that allows the caller to say if they want to allow it. So maybe rather remove this one line.